ecosystem/sep-0006.md

```diff
@@ -463,21 +473,33 @@ The response should be a JSON object like:
       "fee_fixed": 5,
       "fee_percent": 0,
       "min_amount": 0.1,
-      "max_amount": 1000,
+      "max_amount": 10000,
+      "sep12_types": {
+        "default_withdraw_cash": {
+          "description": "used for cash deposits of less than $1,000"
+        },
+        "default_withdraw_bank_transfer": {
+          "description": "used for bank transfer deposits of less than $1,000"
+        },
+        "large_withdraw_cash": {
+          "description": "used for cash deposits of $1,000 or more"
+        },
+        "large_withdraw_bank_transfer": {
+          "description": "used for bank transfer deposits of $1,000 or more"
+        }
+      },
       "types": {
         "bank_account": {
           "fields": {
-              "dest": {"description": "your bank account number" },
-              "dest_extra": { "description": "your routing number" },
-              "bank_branch": { "description": "address of your bank branch" },
-              "phone_number": { "description": "your phone number in case there's an issue" }
+            "dest": { 
+              "description": "your bank account number"
+            }
```

Comment:
I removed the other attributes because SEP-12 accepts these parameters, and we want to encourage anchors to collect as much information via the standardized fields as opposed to the potentially custom fields described here. `dest` simply serves as the identifier for the bank information provided via SEP-12.